tests/ert/unit_tests/test_run_path_creation.py

```diff
@@ -664,41 +677,8 @@ def test_that_runpaths_are_raised_when_invalid(run_path, expected_raise, msg):
         _ = ErtConfig.from_file("config.ert")
 
 
-@pytest.mark.usefixtures("use_tmpdir")
-@pytest.mark.parametrize(
-    "placeholder",
-    ["<ERTCASE>", "<ERT-CASE>"],
```

Comment:
Will we still replace this in runpath?

Reply:
Sorry, this was just erronously removed. Added it back in.